test/operator/test_apply_operator.py

```diff
@@ -8,22 +8,23 @@
 hilbert_space = nk.hilbert.Qubit(3)
 graph = nk.graph.Chain(3, pbc=True)
 s3_representation = graph.space_group_representation(hilbert_space)
-projector = s3_representation.projector(1)
+projector = s3_representation.projector(2)
 
 ising_ham = nk.operator.IsingJax(
     hilbert_space, nk.graph.Chain(hilbert_space.size), 1, 1
 )
 
-operator_list = [ising_ham, projector, s3_representation[1]]
+operator_list = [ising_ham, projector, s3_representation[2]]
 
 
 model = nk.models.RBM(alpha=2)
 sampler = nk.sampler.MetropolisLocal(hilbert_space)
 
 mc_vstate = nk.vqs.MCState(sampler, model, n_samples=2**12, n_discard_per_chain=15)
 fs_vstate = nk.vqs.FullSumState(hilbert_space, model)
+mc_vstate_chunked = nk.vqs.MCState(sampler, model, n_samples=64, n_discard_per_chain=15, chunk_size=32)
 
-vstate_list = [mc_vstate, fs_vstate]
+vstate_list = [mc_vstate, fs_vstate, mc_vstate_chunked]
```

Comment:
can you not add a test case like that, but just add a case in the `test_apply_operator` like
```python
vstate.chunk_size = XYZ
transformed_vstate = nk.vqs.apply_operator(operator, vstate)
assert transformed_vstate.chunk_size == ZZX
```
We don't need to check that the vstate works with chunking. 
just that chunking is set correctly

(other tests elsewhere check that it works with chunking)

Reply:
```suggestion
vstate_list = [mc_vstate, fs_vstate]
```